server/src/integrations/environment/environment.service.ts

```diff
@@ -6,6 +6,7 @@ import { AwsRegion } from './interfaces/aws-region.interface';
 import { StorageType } from './interfaces/storage.interface';
 import { SupportDriver } from './interfaces/support.interface';
 import { LoggerDriver } from './interfaces/logger.interface';
+import { MessageQueueType } from './interfaces/message-queue.interface';
```

Comment:
Since we use "Driver" for Logger and Support on the two lines above, maybe we should use Driver instead of Type?

Reply:
Although I just saw we use Type for storage so we're just not consistent already!